src/dune_rules/melange/melange_stanzas.ml

```diff
@@ -11,7 +11,7 @@ module Emit = struct
     ; libraries : Lib_dep.t list
     ; package : Package.t option
     ; preprocess : Preprocess.With_instrumentation.t Preprocess.Per_module.t
-    ; runtime_deps : Dep_conf.t list
+    ; runtime_deps : Loc.t * Dep_conf.t list
```

Comment:
The location should be come from the atom that introduces the sandbox mode. I guess this is still an improvement for now.

Reply:
I can look into that separately, but having experimented with it in the past, I found that carrying a loc for all elements in the list was a bit heavy for no significant return.